components/Dashboard/candidate/profile/primary-selects.tsx

```diff
@@ -0,0 +1,108 @@
+/* This example requires Tailwind CSS v2.0+ */
+import React from "react";
+import { Listbox, Transition } from "@headlessui/react";
+
+import { Fragment } from "react";
+import { IoIosArrowDown } from "react-icons/io";
+interface selectProps {
+    label: string;
+    selected: any;
+    setSelected: any;
+    data: any;
+    name?: string;
+    height?: string;
+}
+
+function classNames(...classes: any) {
+    return classes.filter(Boolean).join(" ");
+}
+
+ const   PrimarySelect : React.FC<selectProps> = ({
+    label,
+    selected,
+    setSelected,
+    data,
+    name,
+    height,
+}) => {
+    // const [selected, setSelected] = useState(people[3])
+
+    return (
+        <Listbox value={selected} onChange={setSelected}>
+            {({ open }) => (
+                <div className="flex flex-col w-full">
+                    <Listbox.Label className="text-xs text-foreground font-medium dm-sans ">
+                        {label && label}
+                    </Listbox.Label>
+                    <div className=" relative w-full">
+                        <Listbox.Button
+                            className={`${height ? height : "h-[56px] bg-white"
```

Comment:
Change the arbitrary values

```suggestion
                            className={`${height ? height : "h-14 bg-white"
```